release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,62 @@
-| Plugin name | Plugin ID | Version | Category
-
-| Segment | `@janus-idp/backstage-plugin-analytics-provider-segment` | 1.2.0 | Analytics
-
-|Keycloak |`@janus-idp/backstage-plugin-keycloak-backend` |1.7.6 |Identity and access management
-
-|Open Cluster Management (OCM)|`@janus-idp/Backstage-plugin-ocm`|3.5.0|Cluster management
-
-|OCM backend|`@janus-idp/backstage-plugin-ocm-backend`|3.4.6|Cluster management
-
-|Quay|`@janus-idp/backstage-plugin-quay`|1.4.6|Container management
-
-|Quay actions|`@janus-idp/backstage-scaffolder-backend-module-quay`|1.2.1|Container management
-
-|RBAC backend|`@janus-idp/backstage-plugin-rbac-backend`|1.6.3|Security
-
-|RBAC common|`@janus-idp/backstage-plugin-rbac-common`|1.1.0|Security
-
-|Tekton|`@janus-idp/backstage-plugin-tekton`|3.1.3|Cluster management
-
-|Topology|`@janus-idp/backstage-plugin-topology`|1.16.4|Cluster management
-
-|Regex actions|`@janus-idp/backstage-scaffolder-backend-module-regex`|1.2.1|Continuous deployment
-
-|ArgoCD|`@roadiehq/backstage-plugin-argo-cd`|2.3.5|Continuous deployment
-
-|ArgoCD backend|`@roadiehq/backstage-plugin-argo-cd-backend`|2.11.3|Continuous deployment
-
-|API Documentation|`@backstage/plugin-api-docs`|0.9.12|Continuous deployment
-
-|App backend|`@backstage/plugin-app-backend`|0.3.54|Continuous deployment
-
-|Auth backend|`@backstage/plugin-auth-backend`|0.19.4|Security
-
-|Auth node|`@backstage/plugin-auth-node`|0.4.0|Security
-
-|Catalog frontend|`@backstage/plugin-catalog`|1.14.0|Continuous deployment
+=== Configuration change in the GitHub Org catalog provider  
 
-|Catalog backend|`@backstage/plugin-catalog-backend`|1.14.0|Continuous deployment
+The GitHub Org catalog module for the new backend system switched from an obsolete implementation to a new one, that is, `GithubMultiOrgProvider`. This new implementation brings in changes to the default plugin configuration settings.
 
-|Catalog common|`@backstage/plugin-catalog-common`|1.0.17|Continuous deployment
+=== New Scaffolder Identity client
 
-|Catalog Graph|`@backstage/plugin-catalog-graph`|0.2.37|Continuous deployment
+Identity client functioning differs slightly between the one provided by the new backend system Scaffolder plugin, and the one used in the previous release:
 
-|Catalog Import|`@backstage/plugin-catalog-import`|0.10.1|Continuous deployment
+* The previous release uses the `DefaultIdentityClient` function without any specific option
+* New backend system Scaffolder does not use the default identity service, but still uses the deprecated scaffolder-dedicated `buildDefaultIdentityClient` function, which introduces a slight functioning change in the identity client. 
 
-|Catalog React|`@backstage/plugin-catalog-react`|1.8.5|Continuous deployment
+=== Enable all GitLab actions
+{product} {product-version} enables all GitLab actions by default.
 
-|Events backend|`@backstage/plugin-events-backend`|0.2.15|Continuous deployment
+=== Events support redesign in progress for the new backend system
```

Comment:
This, to me, looks more like a known issue than a new feature, and the same for the next section about "RBAC Plugin". I see that it is here under the new backend section, but wouldn't make more sense to move this to a Known issues section? @davidfestal?

Reply:
@davidfestal, I think we should move both these points to the known issues section because they are kind of work-in-progress items, which we want our users to know about, rather presenting them as new features as part of the migration.